metrics/collectors/versions/versions_collector.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 Canonical Ltd
+
+import datetime
+import urllib.request
+
+from metrics.lib.basemetric import Metric
+
+VERSIONS_STATS_URL = "https://people.canonical.com/~platform/desktop/stats/%s/" % datetime.datetime.now().strftime(
+    "%Y-%m-%d"
+)
+
+
+class VersionsMetrics(Metric):
+    def collect(self):
+        """ Collect the sponsoring queue details"""
+        data = []
+
+        known_reports_lst = []
+        with urllib.request.urlopen(VERSIONS_STATS_URL + "reports") as reports:
+            for report in reports.read().decode("utf-8", errors="ignore").split("\n"):
```

Comment:
semi nitpick:

```suggestion
            for report in reports:
                report = report.decode("utf-8", errors="ignore").strip()
                self.log.debug(f"found {report}")
```

(saves loading the whole thing at once)

Reply:
Thanks